DisplaySettings/DisplaySettings.json

```diff
@@ -1166,8 +1208,8 @@
                 ]
             }
         },
-        "getSurroundVirtualizer":{
-            "summary": "(Version 2) Returns the current surround virtualizer boost settings",
+        "getSurroundVirtualizer(v2)":{
```

Comment:
Remove `(v2)` from the method signature. Application calls that include `(v2)` in the method signature will fail.

Reply:
As discussed keeping only version2 methods and removed (v2) from method.